druzhba/avro.py

```diff
@@ -30,8 +31,22 @@ def _avro_format(inp):
     return inp
 
 
+def _redshift_format(inp):
+    if isinstance(inp, str):
+        # The copy command already has TRUNCATECOLUMNS turned on which will do this for us 
+        # however this protects us from the case where a columns value is ridiciously large 
+        # preventing the row from being ingested.
+        return inp[:MAX_VARCHAR_SIZE]
```

Comment:
It may not actually matter in practice, but since this is in a relatively loop it may be better to only do the slicing conditionally? Or do you know if Python is smart enough to not actually make a copy in that case?